### DISCUSSION TITLE
为什么v2.0.2-b4用不了

### DISCUSSION THREAD

**Discussion by LYLYILY** [cangzhang/ChampR#280] [General]
没有连接到英雄联盟。

**Comment by LYLYILY** [comment]
然后勾选点击appley builds也没反应

**Reply by cangzhang** [reply]
https://t.me/champ_r/98

> 各位如果遇到连接不到客户端的问题，请检查自己的操作系统是否是 win10，以及 lol 安装路径是否包含中文。
> 
> 如果是 win10 且包含中文的话，将安装路径改成英文则可解决该问题。
> 
> 如果是 win11，中文路径应该没有问题（本机测试）。
> 
> 感谢 sfzhang 的帮助调查👍，如果有其他问题请各位继续反馈

**Reply by X012C** [reply]
我也是这个问题，我win11，英文路径。这个版本应该不需要放在和LOL同路径下吧。

**Reply by cangzhang** [reply]
@X012C 任何位置都可以。记得已管理员身份运行

**Reply by X012C** [reply]
试过了，用管理员身份运行也不行。

**Reply by cangzhang** [reply]
[champr_log_ver.zip](https://github.com/cangzhang/champ-r/files/11893709/champr_log_ver.zip)

试试这个版本，然后贴一下 log

**Reply by X012C** [reply]
[log.2023-06-28.txt](https://github.com/cangzhang/champ-r/files/11894176/log.2023-06-28.txt)
这个也不行。 @cangzhang

**Reply by cangzhang** [reply]
@X012C 你的路径里是包含中文的吧。从输出看你的安装路径是 `C:/program files (x86)/wegameapps/英雄联盟`

**Reply by X012C** [reply]
感谢帮助，更改了游戏本体路径为英文之后，成功连接上了。 @cangzhang

**Comment by cangzhang** [comment]
https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b5
